### DISCUSSION TITLE
Proposal to give bonus score based on miss count

### DISCUSSION THREAD

**Discussion by Kaerulans** [ppy/osu#21124] [Ideas] [answered]
This is an idea that I've had after reading the discussion in #20717.

One of the problems with the lazer scoring is that once you break your biggest combo, the only way to increase your score is to improve your accuracy, so there's not a lot of incentive to keep playing well and avoid misses.
In an ideal world, we would just account for all your combos throughout a map, but that information is unfortunately unavailable for historic scores, so even if we were to add that data to the database, it's unclear what to do with all the legacy scores.

As I see it, the only way to fix this issue is to somehow incentivize the player to get as few misses as possible.

## Proposal

The current formula for the combo portion of score:

![old](https://user-images.githubusercontent.com/55179197/199974775-aa328869-326f-40a4-a95d-136424de72f8.png)

Proposed formula:

![new](https://user-images.githubusercontent.com/55179197/199975739-cdb7af08-e7c1-4cb6-be1a-bed668fe78f7.png)

Here's how the two formulas scale with achieved combo if you get 0 misses (only sliderbreaks):

Blue is the old formula, red is the new one
![Screenshot from 2022-11-04 15-48-24](https://user-images.githubusercontent.com/55179197/199976529-94a0af35-3299-44e9-9c03-9ff1401e98be.png)

Here's what it looks like with 10 misses:
![Screenshot from 2022-11-04 15-51-35](https://user-images.githubusercontent.com/55179197/199976841-a308a2a3-e0dc-421a-a890-6053dc9f4d6c.png)

Desmos link: https://www.desmos.com/calculator/9c5hfjt9qr

The new formula gives at most a 50% boost to the combo portion compared to the old one, if you sliderbreak in the middle of the map and get 0 misses.

Upside of the proposed formula:

- It gives an incentive for players to get as few misses as possible, even after their combo breaks

Downsides:

- It makes scoring more complicated
- You would lose some score when you get a miss, which may not feel that good

**Comment by smoogipoo** [comment, marked as answer]
... why did you make this a separate discussion thread?

**Reply by peppy** [reply]
Please repost in https://github.com/ppy/osu/discussions/20717 to keep things in one place.
